src/components/ValidationStatus.vue

```diff
@@ -1,113 +1,77 @@
-      <b-form-checkbox
-        data-test="validation-toggle"
-        class="h-100 d-flex align-items-center"
-        v-model="autoValidate"
-        switch
-      >
-        {{ $t('Auto validate') }}
-      </b-form-checkbox>
-
-      <div class="divider"/>
-
-      <div data-test="validation-list-toggle" class="status-bar-container__status" @click="toggleValidationPanel = !toggleValidationPanel; toggleWarningsPanel = false">
-        <div class="d-flex align-items-center">
-          <span class="status-bar-container__status-text">{{ $t('Problems') }} {{ numberOfValidationErrors }}</span>
-          <font-awesome-icon class="status-bar-container__status-icon h-100" :style="{ color: statusColor }" :icon="statusIcon" />
-          <font-awesome-icon class="status-bar-container__status-ellipsis" :icon="ellipsisIcon" />
+      {{ $t('Auto validate') }}
+    </b-form-checkbox>
+
+    <span class="divider" />
+
+    <div v-if="isProblemsPanelDisplayed && numberOfProblems" class="validation-container position-absolute text-left" data-test="validation-list">
+      <div class="validation-container-list d-flex align-items-baseline" v-for="error in errorList" :key="`${error.id}_${error.errorKey}`">
+        <font-awesome-icon
+          class="status-bar-container-status-icon ml-1 mr-2 mt-1"
+          :style="{ color: isErrorCategory(error) ? errorColor : warningColor }"
+          :icon="isErrorCategory(error) ? faTimesCircle: faExclamationTriangle"
+        />
+        <div class="validation-container-list-message">
+          <h6 class="text-capitalize mb-0">{{ error.errorKey }}</h6>
+          <p class="mb-0"><em>{{ error.message }}.</em></p>
+          <p class="mb-0" v-if="error.id"><strong>{{ $t('Node ID') }}:</strong> {{ error.id }}</p>
         </div>
       </div>
 
-      <div class="divider"/>
-
-      <div class="status-bar-container__status" @click="toggleWarningsPanel = !toggleWarningsPanel; toggleValidationPanel = false">
-        <div class="d-flex align-items-center">
-          <span class="status-bar-container__status-text">{{ $t('Warnings') }} {{ warnings.length }}</span>
-          <font-awesome-icon
-            class="status-bar-container__status-icon h-100"
-            :style="{ color: warnings.length > 0 ? warningColor : validColor }"
-            :icon="warnings.length > 0 ? faExclamationTriangle : faCheckCircle"
-          />
-          <font-awesome-icon class="status-bar-container__status-ellipsis" :icon="ellipsisIcon" />
+      <div class="validation-container-list d-flex align-items-baseline" v-for="(warning, index) in warnings" :key="index">
+        <font-awesome-icon class="status-bar-container-status-icon ml-1 mr-2 mt-1" :style="{ color: warningColor }" :icon="faExclamationTriangle" />
+        <div class="validation-container-list-message">
+          <h6 class="text-capitalize mb-0">{{ warning.title }}</h6>
+          <p class="mb-0"><em>{{ warning.text }}.</em></p>
```

Comment:
This structure may change if you switch to using a `dl` element, but I don't think the wrapper `p` element is required. Could you get away with using only the `em` element here?

Another note, following from the previous comment, is that `em` is used for emphasis, and if you're only concerned about applying an italicized style, that should be done with CSS instead.